### PR TITLE
Fix: Fixed crash when switching between tabs

### LINKED ISSUE #11683: Bug: App crash when switching to another tab while items are selected
### Description

The app crashes if you selected a file and switched to another tab while the files are currently selected.

Works fine if you don't select anything before you switch tabs

### Steps To Reproduce

1. Open 2 or more tabs
2. Select a file
3. Switch to another tab

### Requirements

- Fix the crash

### Files Version

2.4.51.0

### Windows Version

10.0.22624.1391

### Log file

```
2023-03-13 02:15:53.0561|ERROR|AppUnhandledException|Object reference not set to an instance of an object.
	System.NullReferenceException: Object reference not set to an instance of an object.
   at Files.App.Actions.Content.ImageEdition.RotateLeftAction.Context_PropertyChanged(Object sender, PropertyChangedEventArgs e)
   at Files.App.Contexts.ContentPageContext.UpdateSelectedItems()
   at Files.App.Contexts.ContentPageContext.Update()
   at Files.App.Contexts.ContentPageContext.Context_Changed(Object sender, EventArgs e)
   at Files.App.UserControls.MultitaskingControl.BaseMultitaskingControl.MultitaskingControl_CurrentInstanceChanged(Object sender, CurrentInstanceChangedEventArgs e)
   at ABI.Microsoft.UI.Xaml.Controls.SelectionChangedEventHandler.Do_Abi_Invoke(IntPtr thisPtr, IntPtr sender, IntPtr e)
--- End of stack trace from previous location ---
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|20_0(Int32 hr)
   at ABI.Windows.ApplicationModel.Core.IUnhandledErrorMethods.Propagate(IObjectReference _obj)
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelperWinUI.<.ctor>b__0_3(Object sender, UnhandledErrorDetectedEventArgs eventArgs)
--- End of stack trace from previous location ---
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelperWinUI.<.ctor>b__0_3(Object sender, UnhandledErrorDetectedEventArgs eventArgs)
   at ABI.System.EventHandler`1.Do_Abi_Invoke[TAbi](Void* thisPtr, IntPtr sender, TAbi args)
   ```

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #11683 

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

File: src/Files.App/Actions/Content/ImageEdition/RotateLeftAction.cs [hunk 1]
```diff
@@ -39,7 +39,7 @@ private void Context_PropertyChanged(object? sender, PropertyChangedEventArgs e)
 		{
 			if (e.PropertyName is nameof(IContentPageContext.HasSelection))
 			{
-				if (context.ShellPage is not null)
+				if (context.ShellPage is not null && context.ShellPage.SlimContentPage is not null)
```
Comment (yaira2) [inline src/Files.App/Actions/Content/ImageEdition/RotateLeftAction.cs:42]: What's the difference between ShellPage and SlimContentPage?

**Review by yaira2 — APPROVED** [review]
LGTM

File: src/Files.App/Actions/Content/ImageEdition/RotateLeftAction.cs (hunk 1 above)
Comment (ferrariofilippo) [reply src/Files.App/Actions/Content/ImageEdition/RotateLeftAction.cs:42]: `ShellPage` is `Modern` or `Column`, `SlimeContentPage` refers to the layout in use
I think they are not synced and so one is updated while the other is not and remains null.
I had found this bug while working on #11673 and I had fixed it there too.

Comment (yaira2) [reply src/Files.App/Actions/Content/ImageEdition/RotateLeftAction.cs:50]: Thanks! We really need to move away from this concept but that's not your fault.

**Pull request merged**